test/suites/secondary_url.bash

```diff
@@ -0,0 +1,41 @@
+SUITE_secondary_url_SETUP() {
+    generate_code 1 test.c
+}
+
+SUITE_secondary_url() {
+    # -------------------------------------------------------------------------
+    TEST "Reject empty url (without config attributes)"
+
+    export CCACHE_SECONDARY_STORAGE="|"
+    $CCACHE_COMPILE -c test.c 2> "${ABS_TESTDIR}/stderr.log"
+    expect_contains "${ABS_TESTDIR}/stderr.log" "must provide an URL"
+
+    # -------------------------------------------------------------------------
+#    TEST "Reject empty url (but with config attributes)"
```

Comment:
The `Util::split_into_views` omits empty entries. That way `|key=value` gets parsed into an vector of one element and the `key=value` argument is interpreted as URL.

Should I make it configurable or could we change default behavior?

Reply:
Thanks for the quick addition.